internal/core/converter/tensorflow/tensorflow_test.go

```diff
@@ -384,57 +493,218 @@ func Test_tensorflow_GetVector(t *testing.T) {
+				feeds:         nil,
+				fetches:       nil,
+				operations:    nil,
+				sessionTarget: "",
+				sessionConfig: nil,
+				options:       nil,
+				graph:         nil,
+				session: &mockSession{
+					RunFunc: func(feeds map[tf.Output]*tf.Tensor, fetches []tf.Output, operations []*tf.Operation) ([]*tf.Tensor, error) {
+						return []*tf.Tensor{nil}, nil
+					},
+				},
+				ndim: 0,
+			},
+			want: want{
+				want: nil,
+				err:  errors.ErrNilTensorTF([]*tf.Tensor{nil}),
+			},
+			checkFunc: defaultCheckFunc,
+		},
+		{
+			name: "failed to cast error: ndim == TwoDim",
+			args: args{
+				inputs: nil,
+			},
+			fields: fields{
+				exportDir:     "",
+				tags:          nil,
+				feeds:         nil,
+				fetches:       nil,
+				operations:    nil,
+				sessionTarget: "",
+				sessionConfig: nil,
+				options:       nil,
+				graph:         nil,
+				session: &mockSession{
+					RunFunc: func(feeds map[tf.Output]*tf.Tensor, fetches []tf.Output, operations []*tf.Operation) ([]*tf.Tensor, error) {
+						tensor, err := tf.NewTensor("test")
+						if err != nil {
+							return nil, errors.New("NewTensor error")
+						}
+						return []*tf.Tensor{tensor}, nil
+					},
+				},
+				ndim: 2,
+			},
+			want: want{
+				want: nil,
+				err:  errors.ErrFailedToCastTF("test"),
+			},
+			checkFunc: defaultCheckFunc,
+		},
+		{
+			name: "failed to cast error: ndim == ThreeDim",
+			args: args{
+				inputs: nil,
+			},
+			fields: fields{
+				exportDir:     "",
+				tags:          nil,
+				feeds:         nil,
+				fetches:       nil,
+				operations:    nil,
+				sessionTarget: "",
+				sessionConfig: nil,
+				options:       nil,
+				graph:         nil,
+				session: &mockSession{
+					RunFunc: func(feeds map[tf.Output]*tf.Tensor, fetches []tf.Output, operations []*tf.Operation) ([]*tf.Tensor, error) {
+						tensor, err := tf.NewTensor("test")
+						if err != nil {
+							return nil, errors.New("NewTensor error")
+						}
+						return []*tf.Tensor{tensor}, nil
+					},
+				},
+				ndim: 3,
+			},
+			want: want{
+				want: nil,
+				err:  errors.ErrFailedToCastTF("test"),
+			},
+			checkFunc: defaultCheckFunc,
+		},
+		{
+			name: "failed to cast error: ndim == default",
+			args: args{
+				inputs: nil,
+			},
+			fields: fields{
+				exportDir:     "",
+				tags:          nil,
+				feeds:         nil,
+				fetches:       nil,
+				operations:    nil,
+				sessionTarget: "",
+				sessionConfig: nil,
+				options:       nil,
+				graph:         nil,
+				session: &mockSession{
+					RunFunc: func(feeds map[tf.Output]*tf.Tensor, fetches []tf.Output, operations []*tf.Operation) ([]*tf.Tensor, error) {
+						tensor, err := tf.NewTensor("test")
+						if err != nil {
+							return nil, errors.New("NewTensor error")
+						}
+						return []*tf.Tensor{tensor}, nil
+					},
+				},
+				ndim: 0,
+			},
+			want: want{
+				want: nil,
+				err:  errors.ErrFailedToCastTF("test"),
+			},
+			checkFunc: defaultCheckFunc,
+		},
```

Comment:
Could you please add test case of the following lines?

1.https://github.com/vdaas/vald/blob/test/internal/tensorflow/internal/core/converter/tensorflow/tensorflow.go#L133-L136
2.https://github.com/vdaas/vald/blob/test/internal/tensorflow/internal/core/converter/tensorflow/tensorflow.go#L142-L146
3. https://github.com/vdaas/vald/blob/test/internal/tensorflow/internal/core/converter/tensorflow/tensorflow.go#L153